### ISSUE TITLE
The progress element should be used to determine the completion of a specific task (WCAG 4.1.2) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#196]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
Page:
Personalizing Canadian Travel Documents landing page

Description:
The progress element should be used to determine the completion of a specific task.

According to the "First Rule of ARIA Use" please use the native HTML <progress> element.
Semantic native HTML elements are readable by sighted viewers, but will also be usable by default by assistive technologies like screen readers, and confer other advantages too.

Note: the accessible name (always should be provided) can come from the associated <label>. Otherwise, use aria-labelledby if a visible label is present or aria-label if a visible label is not present.

For example:
   • <progress value="75" max="100" aria-labelledby="63a32079b75e454dafed1e1e-heading"></progress>
where the aria-labelledby value is an id of the task heading.
*******************************************************************************
